concepts/0430-machine-readable-trust-frameworks/README.md

```diff
@@ -0,0 +1,304 @@
+* What terms of service are standard, odd-but-reasonable, or prohibited?
+
+Many industry groups are exploring these questions, and are building careful documentation of the answers they produce. It is not the purpose of this RFC to duplicate or guide such work. Rather, it's our goal to answer a secondary question:
+
+#### The Question Tackled By This RFC
+
+>How can answers to these questions be represented so they are consumable as artifacts of software policy? 
+
+When we have good answers to this question, we can address feature requests like the following:
+
+#### Desirable Features
+
+* All the software used at a refugee camp &mdash; whether built by the United Nations, by an NGO like Doctors Without Borders, by startups providing identity wallets to individuals, or by military personnel &mdash; has a shared understanding about standards of proof, consent, enforcement, appeal, and privacy.
+* Rules in an ecosystem can be versioned and updated formally, and parties who interact can know unambiguously whether their expectations align. If there is a misalignment, it is easy to identify the particulars.
+* A user of an SSI-centric mobile app can be prompted about whether they'd like to accept as proof credentials issued under the rules of trust framework X (e.g., the trust framework for accredited institutions of higher education in Germany) &mdash; allowing them to accept or reject entire categories of proof without having to track the DIDs and business practices of every individual issuer they might encounter.
+
+#### Sample Data Structure
+
+Trust frameworks generally begin as human-friendly content. They have to be created, reviewed, and agreed upon by experts from various disciplines: legal, business, humanitarian, government, trade groups, advocacy groups, etc. Developers can help by surfacing how rules are (or are not) susceptible to modeling in formal data structures. This can lead to an iterative process, where data structures and human conversations create refinement pressure on each other until the framework is ready for release.
+
+Each problem domain will probably have unique requirements. Therefore, we start with a general trust framework recipe, but plan for extension. We use JSON-LD for this purpose. Here we present a simple example for the problem domain of university credentials in Germany. It manifests just the components of a trust framework that are common across all contexts; additional JSON-LD `@context` values can be added to introduce more structure as needed. 
+
+```jsonc
+{
+    "@context": [
+        // The first context must be this RFC. It defines core properties.
+        "https://github.com/hyperledger/aries-rfcs/concepts/0430-machine-readable-trust-frameworks", 
+        // Additional contexts can be added to extend.
+        "https://kmk.org/uni-accred-trust-fw"
+    ],
+    // name
+    "name": "Universitätsakkreditierung"
+    // version (semver rules apply)
+    "1.0",
```

Comment:
I believe this isn't valid json

```json
{
"1.0"
}
```

Looks like it's red because it seems invalid. Maybe something like 
```json
{
"version":"1.0"
}
```

Reply:
Good catch! Brain moving faster than fingers... :-) I'll fix.

Reply:
Fixed.